docs/contributing/committers.md

```diff
@@ -85,8 +85,8 @@ Summary:
 
 If they accept, then do:
 
-1. Add to Committer team of GitHub Islandora organization ??
-2. Add to Committer team of GitHub Islandora-Labs organization ??
```

Comment:
What does this team do? I don't think committers as a team have any greater access then a regular user, unless granted by... you maybe?

Reply:
I'm sorry, we probably have write permissions. I wasn't thinking clearly. Nevermind this one.